src/test/java/com/epam/tc/hw9api/BoardTest.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.tc.hw9api;
+
+import static com.epam.tc.hw9api.constants.Constants.BOARDS_ENDPOINT;
+import static com.epam.tc.hw9api.specifications.BaseService.notFoundResponseSpecification;
+import static com.epam.tc.hw9api.specifications.BoardService.boardRequestBuilder;
+import static com.epam.tc.hw9api.specifications.BoardService.parseBoard;
+import static com.epam.tc.hw9api.steps.BoardSteps.createBoard;
+import static com.epam.tc.hw9api.steps.BoardSteps.deleteBoard;
+import static com.epam.tc.hw9api.steps.BoardSteps.getDeletedBoard;
+import static com.epam.tc.hw9api.utils.RandomString.generateRandomName;
+import static io.restassured.http.Method.PUT;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import com.epam.tc.hw9api.beans.Board;
+import org.testng.annotations.Test;
+
+public class BoardTest extends BaseTest {
+    Board board;
+
+    @Test
+    public void createBoardTest() {
```

Comment:
Create a GET BOARD step, that returns a BOARD. In your test you 1. Create a board 2. Get this board using GET BOARD step 3. Compare that the name from CREATE request is  equals to the name in GET request. The test in the current state does not check anything

Reply:
done